pyomo/contrib/mindtpy/cut_generation.py

```diff
@@ -153,6 +163,7 @@ def add_oa_cuts(target_model, dual_values, solve_data, config,
         # Equality constraint (makes the problem nonconvex)
         if constr.has_ub() and constr.has_lb() and constr.upper == constr.lower and config.use_dual:
             sign_adjust = -1 if solve_data.objective_sense == minimize else 1
+            # !!THIS SEEMS LIKE A BUG!! - mrmundt #
```

Comment:
@jsiirola - `rhs` has never been assigned anywhere, so the statement `rhs = foo if bar else rhs` doesn't work if the `else` is triggered.

Reply:
@bernalde: thoughts?

Reply:
Absolutely right. This is a bug inherited from the inequality case. The else branch will never be triggered (because of the conditional 2 lines above in 164). A simple fix (you can commit it or we will do it) is simply:
`rhs = constr.lower`

Reply:
@bernalde - I'll commit it, no problem (this and `single_tree`). Thanks for following up!